Bio/Seq.py

```diff
@@ -1464,6 +1464,8 @@ def count_overlap(self, sub, start=0, end=sys.maxsize):
         2
         >>> UnknownSeq(4, character="N").count_overlap("NNNNN")
         0
+        >>> UnknownSeq(4,character="N").count_overlap('NNN',1,2)
```

Comment:
The docstring isn't following PEP8 in terms of spaces after commas - but I think this and most of the other doctests here would be better off in the unit tests as they do little in terms of documentation.